.eslintrc.js

```diff
@@ -62,13 +64,12 @@ module.exports = {
         node: true
       },
       plugins: ['node'],
-      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
```

Comment:
ESLint 6.8+ allows us to use `extends` directly in the overrides configuration, so we no longer need this manual shenanigans.